presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -310,11 +310,23 @@ Java Keystore File Verification
 Verify the password for a keystore file and view its contents using
 :ref:`troubleshooting_keystore`.
 
+SSL debugging for the server
```

Comment:
Title case should be same as in rest of doc. Overall we should change all docs in prestosql but that has to be done in one sweep.